package.json

```diff
@@ -16,7 +16,7 @@
     "build": "npm run lint && tsc -p tsconfig.build.json",
     "buildtests": "npm run lint && tsc -p tsconfig.json",
     "lint": "tslint --project tsconfig.json",
-    "test": "jest",
+    "test": "tsc --noEmit -p . && jest",
```

Comment:
please remove this again, because in the CI it is executed before and otherwise there is the script `buildtests`